src/walletConnectV2Provider.ts

```diff
@@ -200,13 +195,22 @@ export class WalletConnectV2Provider {
         const session = await options.approval();
 
         if (options.token) {
-          const address = this.getAddressFromSession(session);
+          const address = getAddressFromSession(session);
+
+          const selectedNamespace =
+            session.namespaces[WALLETCONNECT_MULTIVERSX_NAMESPACE];
+          const method = selectedNamespace.methods.includes(
+            OptionalOperation.SIGN_NATIVE_AUTH_TOKEN
+          )
+            ? OptionalOperation.SIGN_NATIVE_AUTH_TOKEN
+            : OptionalOperation.SIGN_LOGIN_TOKEN;
+
```

Comment:
If the obsolete login is being used, can we display a WARN message?

Reply:
I think it's best that we add this warning once we have the nativeAuth login flow documented